src/main/java/Bobble.java

```diff
@@ -0,0 +1,104 @@
+                break;
+            case "event":
+                parts = UserInputs[1].split("/from");
+                String[] duration = parts[1].split("/to");
+                tasklist[taskCount] = new Event(parts[0], duration[0], duration[1]);
+                addTaskResponse(tasklist[taskCount], taskCount);
+                taskCount++;
+                break;
+            case "mark":
+                int taskNumber = Integer.parseInt(userInput.substring(5)) - 1;
+                tasklist[taskNumber].setDone(true);
+                markResponse(tasklist[taskNumber]);
+                break;
+            case "unmark":
+                taskNumber = Integer.parseInt(userInput.substring(7)) - 1;
+                tasklist[taskNumber].setDone(false);
+                unmarkResponse(tasklist[taskNumber]);
+                break;
+            default:
+            }
+            userInput = input.nextLine();
+        }
+        goodbye();
+    }
+
+    private static void listResponse(Task[] tasklist,int taskCount) {
+        System.out.println("____________________________________________________________\n" +
+                "Here are the tasks in your list:");
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println((i + 1) + "." + tasklist[i].toString());
+        }
+        System.out.println("____________________________________________________________\n");
+    }
+
+    private static void markResponse(Task tasklist) {
+        System.out.println("Nice! I've marked this task as done:\n" +
+                tasklist.toString() +
+                "\n____________________________________________________________\n");
+    }
+
+    private static void unmarkResponse(Task task) {
+        System.out.println("OK, I've marked this task as not done yet:\n" +
+                task.toString() +
+        "\n____________________________________________________________\n");
+    }
+
+    public static String[] splitUserInput(String input) {
+        String[] userInput = input.split(" ", 2);
+        return userInput;
+    }
+
+    public static void addTaskResponse(Task task, int taskCount) {
+        System.out.println("____________________________________________________________\n" +
+                "Got it. I've added this task: \n  " +
+                task.toString() +
+                "\nNow you have " + (taskCount + 1) + " task(s) in the list." +
+                "\n____________________________________________________________\n");
+    }
+
+    //Greets user
+    public static void start() {
+        System.out.println("____________________________________________________________\n" +
+                "Hello! I'm Bobble.\n" +
+                "What can I do for you?" +
+                "\n____________________________________________________________\n");
+    }
+
+    //Exits
+    public static void goodbye() {
+        System.out.println("____________________________________________________________\n" +
```

Comment:
May want to set a String constant to make it easier to print the horizontal line, or omit it altogether since it is nor required. Will tidy up the code.